tensorflow_datasets/scripts/replace_fake_images.py

```diff
@@ -63,7 +66,7 @@ def rewrite_image(filepath):
 
   image = np.ones_like(image) * color
   image = PIL.Image.fromarray(image)
-  image.save(filepath)
+  image.save(filepath, optimize=True)
```

Comment:
@MadhavEsDios  yes you are right settings optimize will help but when I tried it I observed it decreases size of .jpeg images only but increases size for png format.

Reply:
Thank you for pointing that out I will check file-sizes and confirm what you have observed.

Reply:
Thank you sure

Reply:
Hey, I checked a couple of file-sizes and it looked OK, there was no increase in size for the png format. If you still have concerns, please refer to the "fake_examples" dir in my latest commit which contains the output of the modified script.
Perhaps, you find some anomalies which we can then address sequentially!

Reply:
Please have a look for these files : `tensorflow_datasets/testing/test_data/fake_examples/curated_breast_imaging_ddsm/1.3.6.1.4.1.9590.100.1.2.0004618681.0223128662/1.3.6.1.4.1.9590.100.1.2.0824689759.0062556928`.  Some image sizes increases from 107 to 116 which is small but still increasing in size.

Reply:
I think increase smaller than 1KB are fine.